sandbox/sandbox/src/embedded_executor.rs

```diff
@@ -46,7 +46,15 @@ impl<T> Store<T> {
 
 impl<T> SandboxStore<T> for Store<T> {
     fn new(state: T) -> Self {
-        let engine = Engine::default();
+        let register_len = mem::size_of::<UntypedValue>();
+
+        let mut config = Config::default();
+        config.set_stack_limits(
+            StackLimits::new(1024 / register_len, (1024 * 1024) / register_len, 16384)
```

Comment:
mv magic numbers to const and/or variables above

Reply:
in general, these constants should be in our fork, but we have 4 versions of wasmi in Cargo.lock (see #3214)

Reply:
temp fix is 3f1027548d7f9db011acf9eac856c3f61476bb84